### DISCUSSION TITLE
integrate justdelete.me and accountkiller

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/tosdr.org#109] [Ideas]
http://justdelete.me/sites.json we could just copy this data into our points database once of course, but we could also include this into our build process, then it gets updated every time we deploy the site.

will discuss plan for this (and also for doing the same for accountkiller.com) with @ggrin

**Comment by hugoroy** [comment]
cc @rmlewisuk

**Comment by michielbdejong** [comment]
and here's the account killer link: http://www.accountkiller.com/xml/json/

**Comment by hugoroy** [comment]
MIT as well http://www.accountkiller.com/xml/json/license.txt \o/

**Comment by hugoroy** [comment]
So, what’s the status? we should work on that too.
